src/controllers/expenseController.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { ExpenseService } = require('../services/expensesService');
+const { UserService } = require('../services/usersService');
+
+const expenseService = new ExpenseService();
+const userService = new UserService();
+const getAllExpensesController = (req, res) => {
```

Comment:
Is it necessary to add "Controller" in every function name?

```suggestion
const getAllExpenses = (req, res) => {
```